modules/exploits/multi/http/struts2_namespace_ognl.rb

```diff
@@ -0,0 +1,337 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::EXE
+
+  # Eschewing CmdStager for now, since the use of '\' and ';' are killing me
+  #include Msf::Exploit::CmdStager   # https://github.com/rapid7/metasploit-framework/wiki/How-to-use-command-stagers
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Apache Struts2 Namespace RCE',
```

Comment:
`Struts 2`, please.

Reply:
And perhaps add something about the redirect action?

Reply:
This is starting to feel like word vomit.  :-)  Let's try:
> Apache Struts 2 Namespace Redirect OGNL Injection

Reply:
You forgot "cyber." :joy: